src/QsCompiler/Tests.Compiler/TestCases/TypeParameterResolution.qs

```diff
@@ -0,0 +1,267 @@
+﻿// Copyright (c) Microsoft Corporation.
```

Comment:
nit: The majority of test cases in here don't seem to be related to type parameter resolution, so it may make sense to rename the file (and possibly the namespace).

Reply:
Renamed to PopulateCallGraph, since that is the test category all of these tests fall under.